src/components/walletconnect/WcInput/index.tsx

```diff
@@ -41,17 +42,19 @@ const WcInput = (): ReactElement => {
     [walletConnect],
   )
 
+  useEffect(() => {
+    if (uri.startsWith('wc:')) {
```

Comment:
This code is now repeated. Is it needed?

Reply:
It prevents invalid URIs from the being entered. I've moved it to the search param hook instead.